roles/import-from-url/tasks/provision.yml

```diff
@@ -99,3 +99,12 @@
 
 - name: Provision VM resources
   command: "oc create -f /tmp/vm-resources.yml"
+  
+- name: Relist template broker count
+  block:
+    - name: Get current relist count"
+      command: "oc get clusterservicebroker template-service-broker -o jsonpath='{.spec.relistRequests}'"
+      register: current_relist_count
+    - command: "oc patch clusterservicebroker template-service-broker -p '{\"spec\": {\"relistRequests\": {{ current_relist_count.stdout|int + 1 }} }}'"
+  when: image_type == "template"
```

Comment:
Very nice!  The only concern I have with this logic is that it could be racy with simultaneous use of this APB by multiple users.  That being said there is no better way to achieve this that I can see.  Could you test what happens if you decrement the relist requests field?

Reply:
I know what happens, the apb fails because the patch command will fail. I could add an ignore failure since the relist will happen from whichever one gets there first.